src/components/cards/panel-volunteer/index.tsx

```diff
@@ -3,6 +3,7 @@ import * as React from 'react'
 import * as S from './styles'
 import { LINKS } from 'utils/constants'
 import { useTranslation } from 'gatsby-plugin-react-i18next'
+import { getResizedImgUrl } from '../../../utils/get-resized-img-url'
```

Comment:
Když koukám o pár řádků výš, tak je tam import `utils/constants`, tak to vypadá, že nemusíme uvádět tu plnou relativní cestu? Nevím, jaké jsou v tomhle běžné best practices, ale dává mně to smysl.